react/components/SKUSelector/components/SKUSelector.tsx

```diff
@@ -233,7 +233,10 @@ const variationNameToDisplayVariation = ({
       })
     )
     .filter(Boolean) as DisplayOption[]
-
+    
+    options.sort((a,b)=> {
+      return a.label < b.label ? -1 : a.label > b.label ? 1 : 0;
+    })
```

Comment:
If you want to proceed with the merge, the sort must happen only when specified by a prop. This way, it will not break the behavior of other stores.